src/psyclone/tests/nemo/transformations/openacc/data_directive_test.py

```diff
@@ -306,26 +306,24 @@ def test_kind_parameter(parser):
     assert "copyin(wp)" not in gen_code.lower()
 
 
-def test_no_copyin_intrinsics(parser):
+def test_no_copyin_intrinsics(parser, fortran_reader, fortran_writer):
```

Comment:
`pylint` says: `data_directive_test.py:309:30: W0613: Unused argument 'parser' (unused-argument)`.

Reply:
Oh yes, that's not needed anymore as the frontend creates a parser instance.